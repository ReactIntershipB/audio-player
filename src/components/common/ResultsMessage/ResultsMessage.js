import React from 'react';

export const ResultsMessage = () => {
    return (
        <div className="no-results-container">
            <i className="far fa-sad-tear"></i>
            <p>No Results</p>
        </div>
    );
};
