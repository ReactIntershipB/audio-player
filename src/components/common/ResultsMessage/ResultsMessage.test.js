import React from 'react';
import ResultsMessage from './ResultsMessage';
import renderer from 'react-test-renderer';

describe('ResultsMessage', () => {
  it('to match snapshot', () => {
    const resultsMessage = renderer
    .create(<ResultsMessage />)
    .toJSON();

  expect(resultsMessage).toMatchSnapshot();
  });
});
