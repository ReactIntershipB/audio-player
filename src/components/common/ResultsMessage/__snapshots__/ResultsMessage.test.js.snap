// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultsMessage to match snapshot 1`] = `
<div
  className="no-results-container"
>
  <i
    className="far fa-sad-tear"
  />
  <p>
    No Results
  </p>
</div>
`;
